steps/07-stage.sh

```diff
@@ -19,41 +21,103 @@ sed "s/#VERSION#/${VERSION:-0.0.0.0}/" <"$PATCHES/PDFiumConfig.cmake" >"$STAGING
 
 cp "$SOURCE/LICENSES" "$STAGING/LICENSE"
 cp "$BUILD/args.gn" "$STAGING"
-cp -R "$SOURCE/public" "$STAGING/include"
+cp -R "$SOURCE/public" -T "$STAGING/include"
 rm -f "$STAGING/include/DEPS"
 rm -f "$STAGING/include/README"
 rm -f "$STAGING/include/PRESUBMIT.py"
 
 case "$OS" in
   android|linux)
-    mv "$BUILD/libpdfium.so" "$STAGING_LIB"
+    cp "$BUILD/libpdfium.so" "$STAGING_LIB"
     ;;
 
   mac|ios)
-    mv "$BUILD/libpdfium.dylib" "$STAGING_LIB"
+    cp "$BUILD/libpdfium.dylib" "$STAGING_LIB"
     ;;
 
   wasm)
-    mv "$BUILD/pdfium.html" "$STAGING_LIB"
-    mv "$BUILD/pdfium.js" "$STAGING_LIB"
-    mv "$BUILD/pdfium.wasm" "$STAGING_LIB"
+    cp "$BUILD/pdfium.html" "$STAGING_LIB"
+    cp "$BUILD/pdfium.js" "$STAGING_LIB"
+    cp "$BUILD/pdfium.wasm" "$STAGING_LIB"
     rm -rf "$STAGING/include/cpp"
     rm "$STAGING/PDFiumConfig.cmake"
     ;;
 
   win)
-    mv "$BUILD/pdfium.dll.lib" "$STAGING_LIB"
+    cp "$BUILD/pdfium.dll.lib" "$STAGING_LIB"
     mkdir -p "$STAGING_BIN"
-    mv "$BUILD/pdfium.dll" "$STAGING_BIN"
-    [ "$IS_DEBUG" == "true" ] && mv "$BUILD/pdfium.dll.pdb" "$STAGING_BIN"
+    cp "$BUILD/pdfium.dll" "$STAGING_BIN"
+    [ "$IS_DEBUG" == "true" ] && cp "$BUILD/pdfium.dll.pdb" "$STAGING_BIN"
     ;;
 esac
 
+V_BUILD=0
 if [ -n "$VERSION" ]; then
+  V_BUILD=$(echo "$VERSION" | cut -d. -f3)
   cat >"$STAGING/VERSION" <<END
 MAJOR=$(echo "$VERSION" | cut -d. -f1)
 MINOR=$(echo "$VERSION" | cut -d. -f2)
-BUILD=$(echo "$VERSION" | cut -d. -f3)
+BUILD=$V_BUILD
 PATCH=$(echo "$VERSION" | cut -d. -f4)
 END
-fi
\ No newline at end of file
+fi
+
+# begin conda packaging
+
+case "$OS" in
+  linux)
+    CONDA_HOST="linux-64"
+    ;;
+  mac)
+    CONDA_HOST="osx-64"
+    ;;
+  win)
+    CONDA_HOST="win-64"
+    ;;
+  *)
+    CONDA_HOST="none"
+    ;;
+esac
+
+if [ "$CONDA_HOST" != "none" ] && [ "$LIBC" != "musl" ]; then
+    
+  mkdir -p conda/staging/
+  # use only the build version for conda packages to simplify pinning
+  VERSION=$V_BUILD conda build conda/ --output-folder conda/out/
+  
+  case "$OS-$CPU" in
+    linux-x86)
+      CONDA_TARGET="linux-32"
+      ;;
+    linux-arm64)
+      CONDA_TARGET="linux-aarch64"
+      ;;
+    linux-arm)
+      CONDA_TARGET="linux-armv7l"
+      ;;
+    mac-arm64)
+      CONDA_TARGET="osx-arm64"
+      ;;
+    win-x86)
+      CONDA_TARGET="win-32"
+      ;;
+    win-arm64)
+      CONDA_TARGET="win-arm64"
+      ;;
+    *)
+      CONDA_TARGET="none"
+      ;;
+  esac
+  
+  if [ "$CONDA_TARGET" == "none" ]; then
+    mkdir -p conda/staging/$CONDA_HOST/
+    cp conda/out/$CONDA_HOST/*.tar.bz2 conda/staging/$CONDA_HOST/
+  else
+    conda convert conda/out/$CONDA_HOST/*.tar.bz2 -p $CONDA_TARGET -o conda/out/
+    mkdir -p conda/staging/$CONDA_TARGET/
+    cp conda/out/$CONDA_TARGET/*.tar.bz2 conda/staging/$CONDA_TARGET/
```

Comment:
(note for clarity: the `*` will only find a single file - my original intent here was to build each conda package in the individual build jobs, put it into `conda/staging/` and transfer to the root runner using `upload-artifact`)